pytermgui/markup/parsing.py

```diff
@@ -201,20 +201,21 @@ def tokenize_ansi(  # pylint: disable=too-many-locals, too-many-branches, too-ma
         csi = matchobj.groups()[0:2]
         link_osc = matchobj.groups()[2:4]
 
+        if cursor < start:
+            yield PlainToken(text[cursor:start])
+
```

Comment:
It might be that this being further up causes us to yield a non-plain link closer as if it was plain (though moving it back down didn't seem to fix things)